dev-util/gertty/gertty-1.6.0.recipe

```diff
@@ -38,58 +37,49 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
-PYTHON_PACKAGES=(python39 python310)
-PYTHON_VERSIONS=(3.9 3.10)
-for i in "${!PYTHON_PACKAGES[@]}"; do
-pythonPackage=${PYTHON_PACKAGES[i]}
-pythonVersion=${PYTHON_VERSIONS[$i]}
-eval "PROVIDES_${pythonPackage}=\"\
-	${portName}_$pythonPackage = $portVersion\n\
-	cmd:gertty\
-	\"; \
-REQUIRES_$pythonPackage=\"\
-	haiku\n\
-	cmd:python$pythonVersion\n\
-	alembic_py_$pythonPackage\n\
-	dateutil_$pythonPackage\n\
-	gitpython_$pythonPackage\n\
-	mako_$pythonPackage\n\
-	pbr_$pythonPackage\n\
-	ply_$pythonPackage\n\
-	pyyaml_$pythonPackage\n\
-	requests_$pythonPackage\n\
-	sqlalchemy_$pythonPackage\n\
-	typing_extensions_$pythonPackage\n\
-	urwid_$pythonPackage\n\
-	voluptuous_$pythonPackage\
-	\""
-BUILD_REQUIRES="$BUILD_REQUIRES
+pythonVersion=3.10
+pythonPackage=python${pythonVersion//.}
+
+PROVIDES="
+	$portName = $portVersion
+	cmd:gertty
+	"
+REQUIRES="
+	haiku
+	cmd:python$pythonVersion
+	alembic_py_$pythonPackage
+	dateutil_$pythonPackage
+	gitpython_$pythonPackage
 	pbr_$pythonPackage
+	ply_$pythonPackage
+	pyyaml_$pythonPackage
 	requests_$pythonPackage
+	sqlalchemy_$pythonPackage
+	urwid_$pythonPackage
+	voluptuous_$pythonPackage
+	"
+BUILD_REQUIRES+="
+	pbr_$pythonPackage
 	setuptools_$pythonPackage
-	wheel_$pythonPackage"
-BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	"
+BUILD_PREREQUIRES+="
 	cmd:python$pythonVersion
 	"
-done
+
+# gertty was build as a regular python package instead of a stand-alone util before.
+REPLACES="
+	gertty_python39
+	gertty_python310
+	"
 
 INSTALL()
 {
-	for i in "${!PYTHON_PACKAGES[@]}"; do
-		pythonPackage=${PYTHON_PACKAGES[i]}
-		pythonVersion=${PYTHON_VERSIONS[$i]}
+	python=python$pythonVersion
+	installLocation=$prefix/lib/$python/vendor-packages/
+	export PYTHONPATH=$installLocation:$PYTHONPATH
 
-		python=python$pythonVersion
-		installLocation=$prefix/lib/$python/vendor-packages/
-		export PYTHONPATH=$installLocation:$PYTHONPATH
-		mkdir -p $installLocation
-		rm -rf build
-		$python setup.py build install \
-			--root=/ --prefix=$prefix
+	mkdir -p $installLocation
 
-		packageEntries  $pythonPackage \
-			$prefix/lib/python* \
-			$prefix/bin \
-			$docDir
-	done
+	$python setup.py build install \
+		--root=/ --prefix=$prefix
```

Comment:
Now that this is built only once, maybe move the `build` part to a `BUILD()` function? The `python` variable declaration could be moved out of the function into global scope to not repeat it.

Reply:
It was just a suggestion. You can also leave it as is if this works better.